components/CustomButton.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Button } from "./ui/button";
+import { LucideProps } from "lucide-react";
+
+interface ButtonProps {
+  variant?:
+    | "default"
+    | "secondary"
+    | "outline"
+    | "ghost"
+    | "link"
+    | "destructive";
+  size?: "default" | "sm" | "lg" | "icon";
+  className?: string;
+  label?: string;
+  icon?: React.ComponentType<LucideProps & { className?: string }>;
+  onClick?: () => void;
+}
+
+export const CustomButton = ({
+  label,
+  icon: IconComponent,
+  className,
+  ...props
```

Comment:
Remove this wildcard prop. Let's be explicit with what props we are allowing.

Reply:
Wildcard prop removed